src/app/rendering.js

```diff
@@ -12,6 +12,7 @@ export function render({ component, inside, withClasses }) {
 }
 
 function withClassList(component, classList) {
-  component.classList.add(classList);
+  if (classList != '' && classList != undefined)
```

Comment:
Super! ALEEEeee teraz jak już znaleźliśmy ten problem to znaczy, że było coś pominięte w testach funkcji render. Dobrze dopisać na to test :) Potem test Ci pozwoli refaktorować. A refaktor oznacza też, że bez problemu można warunek uprościć do: `!classList`

Reply:
aaa widzisz pospieszyłem się - zaraz dopiszę teścik i refactor (w sumie nie wiem czemu od razu nie napisałem `!classList`)